.azure_pipelines/olive-aml-ci.yaml

```diff
@@ -0,0 +1,57 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+  paths:
+    exclude:
+    - docs/*
+    - examples/*
```

Comment:
we might still want to run after examples changes since the aml test configs might have been updated

Reply:
![image](https://github.com/user-attachments/assets/241f478e-b2c1-40e8-9407-4db255555031)
this looks like a good enough filter

Reply:
Can you also confirm the pipeline is set up in the publicpackages ado and runs as expected?

Reply:
I've run the pipeline.